scss/base.scss

```diff
@@ -54,3 +54,7 @@ code {
   @extend .govuk-main-wrapper;
   padding-top: govuk-spacing(3);
 }
+
+.moj-filter .govuk-select {
+  min-width: 5.5em;
```

Comment:
Might be worth adding a comment explaining that this is decreased from the default since otherwise `moj-filter` can be narrower than this